util/neural_nets/params.py

```diff
@@ -1 +1,26 @@
 input_shape = (48, 128, 3) # rows, cols, channels
+
```

Comment:
Can we move these neural_net scripts to an actual ros package? They don't have to use ROS code, but that will make them usable via `rosrun` so you could run them from anywhere on your machine.
Could call it rr_nn or rr_ml or whatever makes sense.

Reply:
@ebretl thoughts on this?

Reply:
Issue created